src/site/tests/home/00-required.e2e.spec.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+const path = require('path');
+
+const E2eHelpers = require('platform/testing/e2e/helpers');
+
+function verifyGoogleAnalytics(browser) {
+  const googleAnalyticsCode = fs.readFileSync(
+    path.join(
+      __dirname,
+      '../../assets/js/google-analytics/',
+      `${process.env.BUILDTYPE}.js`,
+    ),
+  );
+
+  browser.execute(
+    () => document.querySelector('[data-e2e="analytics-script"]').innerHTML,
+    [],
+    ({ value: analyticsScriptHtml }) => {
+      const codeExists = analyticsScriptHtml.includes(googleAnalyticsCode);
```

Comment:
Maybe it would be better to check for a script src set to Google instead of the raw JS that creates it, but I also want to check that the GA for the right environment is loaded up, and we don't have a convenient way to do that right now.